src/Rokt-Kit.js

```diff
@@ -138,8 +144,21 @@ var constructor = function () {
 
                 // Locally cache the launcher and filters
                 self.launcher = launcher;
-                self.filters = window.mParticle.Rokt.filters;
-                self.filteredUser = window.mParticle.Rokt.filters.filteredUser;
+
+                var roktFilters = window.mParticle.Rokt.filters;
+
+                if (!roktFilters) {
+                    console.warn('Rokt Kit: No filters have been set.');
```

Comment:
should `roktFilters` be set to an empty object if there are no filters?  you'll have to update anywhere else `self.filters` is referenced to check for it first, like https://github.com/mparticle-integrations/mparticle-javascript-integration-rokt/blob/2bdb5e7803a05b070fa170a7a9807d785263de70/src/Rokt-Kit.js#L96C13-L100 and elsewhere.  do a global search for it